src/preamble.js

```diff
@@ -1886,4 +1886,8 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 #endif
 #endif
 
+#if CYBERDWARF
+var emDebugCDFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
```

Comment:
We don't need `em` prefixes in this file, I don't think. How about `cyberDwarfFile`?